src/App.jsx

```diff
@@ -25,26 +54,37 @@ function App() {
 
       <div className="card">
         <h2 id="birthday-request">
-          When&apos;s your birthday?
+          When&apos;s your next birthday?
```

Comment:
Is there a way you can use an actual apostrophe rather than this encoded one as it makes it difficult to read?